src/console.cpp

```diff
@@ -136,6 +136,79 @@ void CDECL IConsolePrintF(TextColour colour_code, const char *format, ...)
 	IConsolePrint(colour_code, buf);
 }
 
+/**
+ * It is possible to print debugging information to the console,
+ * which is achieved by using this function. You need at least a
+ * level 2 (developer) for debugging messages to show up.
+ * Uses printf() style format, for more information look at IConsolePrint()
+ */
+void CDECL IConsoleDebugF(const char* format, ...)
```

Comment:
Seems to me like there's no need to duplicate all these methods. Should be able to just have one with an extra `CC_foo` parameter that does all the variadic args stuff, all others can just call that (with the warning one doing the developer check first)